catalog/landing-zone/Kptfile

```diff
@@ -0,0 +1,192 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: landing-zone
+packageMetadata:
+  shortDescription: Foundational landing zone blueprint for GCP
+openAPI:
+  definitions:
+    io.k8s.cli.setters.org-id:
+      x-k8s-cli:
+        setter:
+          name: org-id
+          value: "123456789012"
+      description: Organization ID value.
+      type: string
+    io.k8s.cli.setters.billing-account-id:
+      x-k8s-cli:
+        setter:
+          name: billing-account-id
+          value: AAAAAA-BBBBBB-CCCCCC
+      description: Billing account ID
+    io.k8s.cli.setters.group-org-admins:
+      x-k8s-cli:
+        setter:
+          name: group-org-admins
+          value: yakima@google.com
+      description: Group which will be granted organization admin permissions.
+    io.k8s.cli.substitutions.group-org-admins-member:
+      x-k8s-cli:
+        substitution:
+          name: group-org-admins-member
+          pattern: group:${group-org-admins}
+          values:
+          - marker: ${group-org-admins}
+            ref: '#/definitions/io.k8s.cli.setters.group-org-admins'
+    io.k8s.cli.setters.group-billing-admins:
+      x-k8s-cli:
+        setter:
+          name: group-billing-admins
+          value: yakima@google.com
```

Comment:
should we swap this default email with a non existent one?

Reply:
Good idea, done.